src/main/features/response/views/confirmation/full-defence-steps.njk

```diff
@@ -2,7 +2,29 @@
 
 {% set claimantName = claim.claimData.claimant.name %}
 
-{% if claim.response.freeMediation === FreeMediationOption.YES %}
+{% if claim.response.defenceType === DefenceType.ALREADY_PAID %}
+  {% if claim.response.freeMediation === FreeMediationOption.YES %}
+    <p>{{ t('We’ll contact you when {{ claimant }} responds, to tell you what to do next.', { claimant: claim.claimData.claimant.name }) }}</p>
+    <p>{{ t('If {{ claimant }} accepts your response the claim will be settled.', { claimant: claim.claimData.claimant.name }) }}</p>
+    <p>{{ t('If {{ claimant }} rejects your response we’ll ask them to try mediation. If they agree, we’ll contact you to arrange an appointment.', { claimant: claim.claimData.claimant.name }) }}</p>
+    <p>{{ t('If they reject mediation the court will review the case. You might have to go to a hearing.') }}</p>
+  {% else %}
+    <p>{{ t('If {{ claimant }} accepts your response the claim will be settled. We’ll contact you when they respond.',
+        { claimant: claim.claimData.claimant.name }) }}</p>
+    <p>{{ t('If they reject your response the court will review the case. You might have to go to a hearing.') }}</p>
+    <p>{{ t('We’ll contact you if we set a hearing date to tell you how to prepare.') }}</p>
+  {% endif %}
+{% elif claim.response.directionsQuestionnaire.selfWitness %}
+  <p>{{ t('We’ll contact you when {{ claimantName }} responds, to tell you what to do next.', { claimantName: claimantName }) | safe }}</p>
+  <p>{{ t('If {{ claimantName }} accepts your response the claim will be settled.', { claimantName: claimantName }) | safe }}</p>
```

Comment:
Why is `| safe` needed?

Reply:
forgot to remove them after copying. removed now